src/fieldPresets/checkbox.js

```diff
@@ -1,10 +1,10 @@
 export default {
   valuePropName: 'checked',
   initialValue: false,
-  mapPropsToField: ({ props: { checked }, fieldRecord }) => ({
+  mapPropsToField: ({ props: { checked, initialValue }, fieldRecord }) => ({
     ...fieldRecord,
     type: 'checkbox',
-    checked: !!checked,
+    checked: !!initialValue || !!checked,
```

Comment:
It looks you don't have your fork up-to-date, so these merged changes appear in the diff. Please do `git pull --rebase` on the `master`, and `git rebase master` in your feature branch.

Otherwise looks great.

Reply:
Oh... my bad